src/stream/quic_lb/ngx_stream_quic_lb_retry_service.c

```diff
@@ -154,313 +154,529 @@ ngx_stream_quic_lb_gen_and_send_retry_packet(ngx_connection_t *c,
 
 
 static ngx_int_t
-ngx_stream_quic_lb_validate_ngx_quic_model_token(ngx_connection_t *c,
-    ngx_quic_header_t *pkt, u_char *key)
+ngx_stream_quic_lb_validate_quic_lb_model_token(ngx_connection_t *c,
+    ngx_quic_header_t *pkt, ngx_quic_lb_conf_t *qconf)
 {
-    int                      len, tlen, iv_len;
-    u_char                  *iv, *p, *data;
-    ngx_msec_t               msec;
-    EVP_CIPHER_CTX          *ctx;
-    const EVP_CIPHER        *cipher;
-    struct sockaddr_in      *sin;
-#if (NGX_HAVE_INET6)
-    struct sockaddr_in6    *sin6;
-#endif
-    u_char                  tdec[NGX_QUIC_MAX_TOKEN_SIZE];
+    size_t                          len;
+    ngx_int_t                       res, i;
+    const EVP_CIPHER               *cipher;
+    ngx_quic_secret_t               s;
+    ngx_str_t                       token_body_plaintext;
+    ngx_str_t                       token_body_enc;
+    ngx_str_t                       aad;
+    ngx_str_t                       plaintext_buf;
+    ngx_str_t                       key_seq;
```

Comment:
1. define key seq as uint8_t will be more suitable
2. key seq should used when select key

Reply:
fixed